data/events/2023-new-york-city.yml

```diff
@@ -0,0 +1,130 @@
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
+  - name: location
+  - name: registration
+  - name: program
+  - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Sumit Agarwal"
+    employer: "Broadridge Financial Solutions"
+    image: "sumit-agarwal.jpg"
+    twitter: "aga_sumit"
+    linkedin: "https://www.linkedin.com/in/agarwalsumitfsdevconsultant/"
+  - name: "Ceren Ercen"
+    employer: "Grubhub/Seamless"
+    image: "ceren-ercen.jpg"
+  - name: "Russ Kaehler"
+    employer: "Memorial Sloan Kettering Cancer Center"
+    image: "russ-kaehler.jpg"
+    linkedin: "https://www.linkedin.com/in/russ-kaehler-514187110"
+  - name: "Matt Kuritz"
+    employer: "The Farmer's Dog"
+    image: "matt-kuritz.jpg"
+    github: "kuritz"
+    twitter: "_kuritz"
+  - name: "Priyanka Rao"
+    image: "priyanka-rao.jpg"
+    linkedin: "https://www.linkedin.com/in/priyanka-rao"
+    twitter: "pri_posterous"
+  - name: "Adarsh Shah"
+    image: "adarsh-shah.jpg"
+    employer: "shahadarsh.com"
+    website: "http://shahadarsh.com"
+    twitter: "shahadarsh"
+    github: "shahadarsh"
+  - name: "Stephen Thomas"
+    employer: "iconectiv"
+    linkedin: "https://www.linkedin.com/in/steve-thomas-4746bb/"
+    image: "stephen-thomas.jpg"
+  - name: "Matthew Titmus"
+    employer: "Flatiron Health"
+    linkedin: "https://www.linkedin.com/in/matthew-titmus/"
+    github: "clockworksoul"
+    image: "matthew-titmus.jpg"
+  - name: "Stefana Muller"
+    employer: "2nd Watch"
+    image: "stefana-muller.jpg"
+    linkedin: "https://www.linkedin.com/in/stefanamuller"
+    twitter: "stefanamuller"
+
+organizer_email: "new-york-city@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: arresteddevops
+    level: community
+
+sponsors_accepted: "yes" # Whether you want "Become a XXX Sponsor!" link
```

Comment:
We might want to set this 'no' for now, deploy preview would help confirm what this does